src/OpenTelemetry.PersistentStorage.FileSystem/README.md

```diff
@@ -14,40 +14,52 @@ exporters to improve the reliability of data delivery.
 dotnet add package OpenTelemetry.PersistentStorage.FileSystem
 ```
 
-## Basic Usage
+## Usage
 
 ### Setup FileBlobProvider
 
 ```csharp
-using var persistentBlobProvider = new FileBlobProvider("test");
+using var persistentBlobProvider = new FileBlobProvider("@C:\temp");
 ```
 
+#### Configuration
+
 Following is the complete list of configurable options that can be used to set
 up FileBlobProvider:
```

Comment:
```suggestion
Following is the complete list of parameters the `FileBlobProvider` constructor
accepts to control its configuration:
```

Kind of a mouthful but I wanted to point out they are ctor parameters. It wasn't clear to me at first because they were being called "options" which is usually a class.